src/table/utils/handle-accessibility.js

```diff
@@ -84,6 +84,7 @@ export const handleFocusoutEvent = (evt, shouldRefocus, keyboard) => {
   if (keyboard.enabled && !evt.currentTarget.contains(evt.relatedTarget) && !shouldRefocus.current) {
     evt.currentTarget.querySelector('#sn-table-announcer--01').innerHTML = '';
     evt.currentTarget.querySelector('#sn-table-announcer--02').innerHTML = '';
+    // Blur the table but not focus its parent element
     keyboard.blur(false);
```

Comment:
is it conflict with https://github.com/qlik-oss/sn-table/pull/564/files#diff-3f227d6a51355e588ecb2a10cd87fc83d3c270d50b08fe3cb560baa979976aeaR36?

Reply:
what do you mean? They should do different things in these two cases

Reply:
when nebula handles keyboard navigation and is not in selection mode, pressing the escape key to tell Nebula to relinquish the table's focus to its parent element. Both keyboard.blur(true) and keyboard.blur(false) run.